### PR TITLE
Page update check

### PR DESCRIPTION
**Pull request by lolamtisch**

### CODE REVIEW

File: src/code/03_handleAnime.js [hunk 1]
```diff
@@ -65,7 +65,7 @@
             $('.lastOpen').remove();
             $.episodeListSelector().each(function( index ) {
                 if(listType == 'anime'){
-                    if(con == console){
+                    if(debug){
```
Comment (dinosw) [inline src/code/03_handleAnime.js:68]: Shouldn't this be "debugging" or "debug = 1"?

File: src/code/03_handleAnime.js [hunk 2]
```diff
@@ -81,7 +81,7 @@
                         }
                     }
                 }else{
-                    if(con == console){
+                    if(debug){
                         $(this).after('   Chapter: '+urlToChapter($(this).episodeListElementHref()));
```
Comment (dinosw) [inline src/code/03_handleAnime.js:85]: Shouldn't this be "debugging" or "debug = 1"?

File: src/code/11_miniMAL.js [hunk 3]
```diff
@@ -489,18 +529,19 @@
...
                 settingsUI += materialCheckbox(episodeInfoImage,'episodeInfoImage','Image'+getTooltip('If enabled, the episode-image from MAL will be displayed in the Episode Hoverinfo.'));
                 settingsUI += materialCheckbox(episodeInfoSubtitle,'episodeInfoSubtitle','Alternative Title'+getTooltip('If enabled, the alternative title for the episode, will be displayed in the Episode Hoverinfo. Example using the anime "Fate/Apocrypha":<br>Title: "Apocrypha: The Great Holy Grail War"<br>Subtitle: "Gaiten: Seihai Taisen (外典:聖杯大戦)"'));
                 settingsUI += '</div>';
 
-                settingsUI += '<div class="mdl-cell mdl-cell--12-col mdl-shadow--4dp">\
+                settingsUI += '<div class="mdl-cell mdl-cell--6-col mdl-cell--8-col-tablet mdl-shadow--4dp">\
                             <div class="mdl-card__title mdl-card--border">\
                                 <h2 class="mdl-card__title-text">ETC</h2>\
                                 </div>';
+                settingsUI += materialCheckbox(debugging,'debugging','Debugging ');
```
Comment (dinosw) [inline src/code/11_miniMAL.js]: settingsUI += materialCheckbox(debugging,'debugging','Debugging');

**Review by dinosw — COMMENTED** [review]
I have read through the changes, but it was from my phone. I'll take a closer look soon 👍

File: src/code/03_handleAnime.js (hunk 1 above)
Comment (lolamtisch) [reply src/code/03_handleAnime.js:68]: `if` checks if something is true or false.
As 1 is the same as using true, there is no need to check explicitly for `1`.

Comment (dinosw) [reply src/code/03_handleAnime.js:68]: 👍

**Review by dinosw — APPROVED** [review]
I think it is good to go ;-)

**Pull request merged**